test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.cs

```diff
@@ -24,22 +25,22 @@ public class HubConnectionTests
         {
             var connection = new Mock<IConnection>();
             connection.SetupGet(p => p.Features).Returns(new FeatureCollection());
-            connection.Setup(m => m.StartAsync()).Returns(Task.CompletedTask).Verifiable();
+            connection.Setup(m => m.StartAsync(new CancellationToken())).Returns(Task.CompletedTask).Verifiable();
```

Comment:
Why are you passing them.

Reply:
I was getting a weird error that said something about not being able to use default values...

Reply:
Use `It.IsAny<CancellationToken>()`?